src/EditorFeatures/Test2/Rename/InlineRenameTests.vb

```diff
@@ -1303,6 +1303,7 @@ class C
 
                 Await editHandler.ApplyAsync(
                     workspace,
+                    workspace.CurrentSolution,
                     workspace.CurrentSolution.GetDocument(workspace.Documents.Single().Id),
```

Comment:
💭 Does `ApplyAsync` need the solution if we have the document?

Reply:
so the Document is the Document we have at hte *Application time*, not at the time when the lightbulbs were computed.  That's the core issue here that hte workspace has potentially slightly moved forward.  The PR ensures we capture the solution at hte time the lightbulbs are actually created so we can determine what textual changes the lightbulb action wanted to make so we can still try to apply those text changes in that event.